tests/test_add_scenario.py

```diff
@@ -0,0 +1,30 @@
+from click.testing import CliRunner
+from pathlib import Path
+import tempfile
+import shutil
+import toml
+import os
+
+from PyDSS.cli.add_scenario import build_scenario
+
+PYDSS_PROJECT = Path(__file__).parent / "data" / "project"
+MAPPING_FILE = Path(__file__).parent / "data" / "add_scenario" / "controller_map.toml"
+
+def test_add_scenario():
```

Comment:
If you want to save some logic, you can use this built-in fixture:
```
def test_add_scenario(tmp_path):
```
`tmp_path` is a `Path` directory that will be deleted for you on exit.

Reply:
I will just keep it as is